metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -266,9 +294,18 @@ void main() {
     test(
       ".pop() sets the previous rote configuration to current configuration",
       () {
-        final expectedConfiguration = DefaultRoutes.dashboard;
+        final notifier = NavigationNotifier(
+          pageFactory,
+          pageParametersFactory,
+          const MetricsPageRouteConfigurationFactory(),
+          navigationState,
+        );
+
+        notifier.handleAppInitialized(isAppInitialized: true);
         notifier.handleAuthenticationUpdates(isLoggedIn: true);
 
+        final expectedConfiguration = DefaultRoutes.dashboard;
```

Comment:
✅ 
Preparing expected values is mostly related to the first A (arrange) in the AAA principle we try to follow. Even if we try to apply the given-when-then approach, the expected values are supposed to be given.